src/test/java/com/epam/dataservice/DaytimeTypeServiceTest.java

```diff
@@ -18,31 +18,66 @@ public static void init() {
 	}
 
 	@Test
-	public void testGetDaytimeType() {
+	public void testGetDaytimeType_night_boundCase_1() {
```

Comment:
I like this separated tests here. Very much :) 
One minor recommendation here - try to not use indexes in test names. You still can make it more meaningful without it. As example this test can be called "testGetDaytimeTypeNightLowerBound" or even better - when_zero_oclock_then_return_night.